lib/cli/interactive-setup/aws-credentials.js

```diff
@@ -86,33 +178,141 @@ const steps = {
       )}\n`
     );
   },
+  // TODO: TESTS
+  handleProviderCreation: async ({ configuration: { org: orgName } }) => {
+    const providersUrl = `https://app.serverless.com/${orgName}/settings/providers?source=cli`;
+    openBrowser(providersUrl);
+    // TODO: CHANGE MESSAGING
+    process.stdout.write('Waiting for creation of provider...\n');
+
+    let onEvent;
+
+    const p = new Promise((resolve) => {
+      let inquirerPrompt;
+
+      const timeoutDuration = 1000 * 60; // 1 minute
+      const showSkipPromptTimeout = setTimeout(() => {
+        inquirerPrompt = inquirer.prompt({
+          message: '\n Press Enter to continue without setting up provider',
+          name: 'skipProviderSetup',
+        });
+
+        inquirerPrompt.then(() => resolve(null));
```

Comment:
Not really - do you have anything specific in mind?

Reply:
No, just wanted to confirm, there's no way it can fail. But I imagine there isn't. 

User can either press enter or Ctrl+C, which I assume Inquirer doesn't handle